lib/ssl/test/tls_1_3_version_SUITE.erl

```diff
@@ -529,3 +602,42 @@ create_bad_client_certfile(NewClientCertFile, ClientOpts) ->
     ClientOTPTbsCert = ClientOTPCert#'OTPCertificate'.tbsCertificate,
     NewClientDerCert = public_key:pkix_sign(ClientOTPTbsCert, Key),
     ssl_test_lib:der_to_pem(NewClientCertFile, [{'Certificate', NewClientDerCert, not_encrypted}]).
+
+test_rsa_pcks1_cert(SHA, COpts, SOpts, Config) ->
+    #{client_config := ClientOpts,
+      server_config := ServerOpts} = public_key:pkix_test_data(#{server_chain => #{root => root_key(SHA),
+                                                                                   intermediates => intermediates(SHA, 1),
+                                                                                   peer => peer_key(SHA)},
+                                                                 client_chain => #{root => root_key(SHA),
+                                                                                   intermediates => intermediates(SHA, 1),
+                                                                                   peer => peer_key(SHA)}}),
+    ssl_test_lib:basic_test(COpts ++ ClientOpts, SOpts ++ ServerOpts, Config).
+
+root_key(SHA) ->
+    %% As rsa keygen is not guaranteed to be fast
```

Comment:
I'm not sure what comment above relates to.
If it explaining why hardcoded RSA key is used, maybe would be better to include that in single placed comment for `ssl_test_lib:hardcode_rsa_key` rather than having it in every place when this function is callled?

Reply:
Comment is copy pasted. Yes it means that generating rsa keys on the fly might some odd time be very time consuming so of we do it for all test it is bound to happen sooner or later giving flaky test cases.